### ISSUE TITLE
Magnitude concentrated on 18~26 with variable dataset and non-variable dataset

### ISSUE THREAD

**Issue by BossunWang** [IrvingMeng/MagFace#7]
Hi, Expert
Nice work to embedded magnitude into latent space of features.
Then I train on masked and non-masked MS1M, and test on LFW, CFP, Megaface
But I wonder the magnitude without any discrimination on testing dataset
May you have any suggestion about this situation?

[magnitude.jpg](https://drive.google.com/file/d/1QhL9XnHuNnGKzXPn402abAsLK-sfRUUt/view?usp=sharing)

**Comment by IrvingMeng** [comment]
Hi, could you provide more details about the training/test configs?

**Comment by BossunWang** [comment]
Thanks reply,
I got more information recently about this,
when I use Arcface loss training from scratch, then fine-tuned by MagFace, I found out this issue.
In fact, the magnitude did't make any changes when use Arcface first and MagFace last.
But training from scratch by MagFace same as paper used, it's OK, the magnitude make variation with different dataset.
May you have occur this situation or some issue like this?

**Comment by IrvingMeng** [comment]
Hi. If you want to finetune your model with MagFace, the parameters in the paper may not always work. 
I add some recommended steps in the Readme (check **Tips for MagFace**). These steps are already verified in our applications and hope they can be helpful to you.

Feel free to contact me if you have any questions :).

**Comment by BossunWang** [comment]
OK, thanks your advice, I'll try it.
If any other idea about this extend issue, I would like to mail to your outlook(IrvingMeng@outlook.com), is it OK?

**Comment by IrvingMeng** [comment]
You can  contact me at WeChat (IrvingMeng), FaceBook (ustcmq) or Email (IrvingMeng@outlook.com), whichever is most convenient.
